layers/+lang/php/packages.el

```diff
@@ -39,6 +41,20 @@
 (defun php/post-init-flycheck ()
   (add-hook 'php-mode-hook 'flycheck-mode))
 
+(defun php/init-geben ()
+  (use-package geben
+    :defer t))
```

Comment:
Probably should add  (setq geben-display-window-function (quote popwin:display-buffer))
as well.